docs/userguide/datafiles.txt

```diff
@@ -89,19 +130,38 @@ the ``setup.py`` ``package_data`` list is updated before calling ``setup.py``.
 
 (Note: although the ``package_data`` argument was previously only available in
 ``setuptools``, it was also added to the Python ``distutils`` package as of
-Python 2.4; there is `some documentation for the feature`__ available on the
-python.org website.  If using the setuptools-specific ``include_package_data``
+Python 2.4; there is `some documentation for the `feature available on the
```

Comment:
This syntax rewrite looks incorrect. There's a backtick in front of "some" but also "feature", so there's an odd number of backticks on this paragraph. Also, the hyperlink is missing a closing `>`.